### PR TITLE
Fix for browser compatibility

### PR DESCRIPTION
**Pull request by jojobyte**
Fix related to Vite Testing

See https://github.com/jojobyte/dashsight-vite-test/commit/0b514b656b918980e8e47bb0a673296b4ba4076d

### CODE REVIEW

File: lib/crowdnode.js [hunk 1]
```diff
@@ -18,15 +18,18 @@ CrowdNode._dashApi = Dash.create({ insightApi: CrowdNode._insightApi });
 
 CrowdNode.main = {
   baseUrl: "https://app.crowdnode.io",
+  knowledge: "https://knowledge.crowdnode.io",
```
Comment (coolaj86) [inline lib/crowdnode.js:21]: Will you open a separate PR for the knowledge base URL addition please? I'll be happy to merge that immediately.

File: lib/crowdnode.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ let CrowdNode = module.exports;
 const DUFFS = 100000000;
 
 let Dash = require("./dash.js");
-let Dashcore = require("@dashevo/dashcore-lib");
+let Dashcore = require("@dashevo/dashcore-lib/dist/dashcore-lib.min.js");
```
Comment (coolaj86) [inline lib/crowdnode.js:10]: I think this would break node. Will need to test.

File: tsconfig.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    /* Visit https:
+    /* Visit https: */
```
Comment (coolaj86) [inline tsconfig.json:3]: Looks like this line is junk that should be removed.

File: lib/crowdnode.js (hunk 1 above)
Comment (jojobyte) [reply lib/crowdnode.js:21]: Done, see https://github.com/dashhive/crowdnode.js/pull/40

File: tsconfig.json (hunk 3 above)
Comment (jojobyte) [reply tsconfig.json:3]: Done, see https://github.com/dashhive/crowdnode.js/pull/41

File: lib/crowdnode.js (hunk 2 above)
Comment (jojobyte) [reply lib/crowdnode.js]: Check out the latest changes, see if those are acceptable.

**Review by coolaj86 — APPROVED** [review]
👍

**Pull request merged**